ovs/matchparser.go

```diff
@@ -243,23 +243,23 @@ func parseMACMatch(key string, value string) (Match, error) {
 
 // parseCTState parses a series of connection tracking values into a Match.
 func parseCTState(value string) (Match, error) {
-	if len(value)%4 != 0 {
-		return nil, errors.New("ct_state length must be divisible by 4")
-	}
-
-	var buf bytes.Buffer
-	var states []string
-
-	for i, r := range value {
-		if i != 0 && i%4 == 0 {
-			states = append(states, buf.String())
-			buf.Reset()
-		}
-
-		_, _ = buf.WriteRune(r)
-	}
-	states = append(states, buf.String())
-
+	if strings.Contains(value, "|") {
+		value = strings.ReplaceAll(value, "|", " ")
+	} else if strings.Contains(value, "+") || strings.Contains(value, "-") {
+		value = strings.ReplaceAll(value, "+", " +")
+		value = strings.ReplaceAll(value, "-", " -")
+		value = strings.Trim(value, " ")
+	} else {
+		// There are two valid format for ct_state:
+		// `ct_state=+est+trk` and `ct_state=est|trk`
+		return nil, errors.New("ct_state format is invalid")
+	}
+
+	// The expected output is:
+	// "est|trk|dnat" => "est trk dnat"
```

Comment:
I'd argue that even this is should be 
```
// "est|trk|dnat" => "+est +trk +dnat"
```

So that the `ConnectionTrackingState()` is consistently representing the same thing.